src/test/datascience/jupyter/kernels/installationPrompts.vscode.test.ts

```diff
@@ -61,7 +61,8 @@ suite('DataScience Install IPyKernel (slow) (install)', () => {
         vscodeNotebook = api.serviceContainer.get<IVSCodeNotebook>(IVSCodeNotebook);
     });
 
-    setup(async () => {
+    setup(async function () {
+        console.log(`Start test ${this.currentTest?.title}`);
```

Comment:
Added this as its a little difficult to figure out with all of the logging when tests start & when they end, as we have a lot of other stuff in setup & teardown (& sometimes we have additoinal logging that leaks from other tests - due to stuff not getting disposed properly)